### DISCUSSION TITLE
JAMPER T PRO V 2

### DISCUSSION THREAD

**Discussion by c0ut3** [EdgeTX/edgetx#4894] [Q&A]
Hello, there is a problem on jumper t pro v 2, I can't enter the expresslrs menu, just endless loading

**Comment by pfeerick** [comment]
Is the ELRS module turned on? I.e. MDL -> Model Setup -> Internal RF ->
ELRS (not OFF)... Goto External RF instead of you are using a nano bay
module.

**Reply by c0ut3** [reply]
![photo_2024-04-18_16-36-49](https://github.com/EdgeTX/edgetx/assets/167347560/7b7db873-c81f-4195-90fb-11760af2613f)
if you're talking about this then it's on

**Reply by c0ut3** [reply]
It did not help

**Comment by 3djc** [comment]
400k is a wrong speed for internal module ? Change it to 921k
